src/tablib/formats/_sql.py

```diff
@@ -0,0 +1,72 @@
+"""Tablib - SQL INSERT Export Support."""
+import datetime
+import decimal
+
+from ..exceptions import UnsupportedFormat
+
+
+class SQLFormat:
+    """Export Dataset rows as SQL INSERT statements."""
+    title = 'sql'
+    extensions = ('sql',)
+
+    @staticmethod
+    def _render_literal(value):
+        """Render a Python value as an SQL literal."""
+        if value is None:
+            return 'NULL'
+        if isinstance(value, bool):
+            return 'TRUE' if value else 'FALSE'
+        if isinstance(value, int) and not isinstance(value, bool):
+            return str(value)
+        if isinstance(value, decimal.Decimal):
+            return str(value)
```

Comment:
No need to repeat the bool check, and can combine two of these:
```suggestion
        if isinstance(value, bool):
            return 'TRUE' if value else 'FALSE'
        if isinstance(value, (int, decimal.Decimal):
            return str(value)
```